### PR TITLE
Cleaning up before using this at GIDS

### PR DESCRIPTION
**Pull request by alpinegizmo**
Goals:

- Builds cleanly (no RAT errors)
- Remove distracting chunks of code that aren't being used
- A bit of renaming to clarify what things are
- Make it possible to run the Java side of things in an IDE (w/o docker)

Still to do:

- Add more explanations to the README
- Describe the exercise(s)

### CODE REVIEW

**Comment by alpinegizmo** [comment]
@sjwiesman A review would be most helpful.

File: statefun-workshop-io/src/main/java/com/ververica/statefun/workshop/io/local/ConfirmedFraudSource.java [hunk 1]
```diff
@@ -22,13 +22,13 @@
 
 import java.util.concurrent.TimeUnit;
 
-public class ConfirmedTransactionSource extends RichSourceFunction<ConfirmFraud> {
+public class ConfirmedFraudSource extends RichSourceFunction<ConfirmFraud> {
```
Comment (sjwiesman) [inline statefun-workshop-io/src/main/java/com/ververica/statefun/workshop/io/local/ConfirmedFraudSource.java:25]: 👍

**Review by sjwiesman — APPROVED** [review]
+1 This looks good to me, not sure if you want to merge it now or push additional updates to this branch

**Review by sjwiesman — APPROVED** [review]
LGTM

**Pull request merged**